packages/generator-one-app-module/generators/app/templates/intl-child-module/test-setup.js

```diff
@@ -0,0 +1,2 @@
+require('amex-jest-preset-react/jest-setup');
```

Comment:
You will need to set up jest-schema in here once you update amex-jest-preset-react. Do so like https://github.com/americanexpress/amex-jest-preset/blob/v5.0.1/jest-setup.js

Reply:
https://github.com/americanexpress/one-app-cli/pull/25#discussion_r382271966